internal/services/compute/orchestrated_virtual_machine_scale_set.go

```diff
@@ -449,7 +449,6 @@ func orchestratedVirtualMachineScaleSetPublicIPAddressSchema() *pluginsdk.Schema
 				"sku_name": {
 					Type:         pluginsdk.TypeString,
 					Optional:     true,
-					Default:      fmt.Sprintf("%s_%s", string(compute.PublicIPAddressSkuNameBasic), string(compute.PublicIPAddressSkuTierRegional)),
```

Comment:
The default values cause some tests to fail
------- Stdout: -------
=== RUN   TestAccOrchestratedVirtualMachineScaleSet_basicPublicIP
=== PAUSE TestAccOrchestratedVirtualMachineScaleSet_basicPublicIP
=== CONT  TestAccOrchestratedVirtualMachineScaleSet_basicPublicIP
testcase.go:110: Step 1/2 error: Error running apply: exit status 1
Error: updating NAT Gateway Association for Subnet: (Name "acctsub-220824145028554777" / Virtual Network Name "acctvn-220824145028554777" / Resource Group "acctestRG-OVMSS-220824145028554777"): network.SubnetsClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="SubnetWithNatGatewayAndBasicSkuResourceNotAllowed" Message="NAT Gateway /subscriptions/*******/resourceGroups/acctestRG-OVMSS-220824145028554777/providers/Microsoft.Network/natGateways/acctng-220824145028554777 cannot be deployed on subnet containing Basic SKU Public IP addresses or Basic SKU Load Balancer. NIC /subscriptions/*******/resourceGroups/acctestRG-OVMSS-220824145028554777/providers/Microsoft.Network/networkInterfaces/TestNetworkProfile-220824145028554777-5c526597/ipConfigurations/TestIPConfiguration in subnet /subscriptions/*******/resourceGroups/acctestRG-OVMSS-220824145028554777/providers/Microsoft.Network/virtualNetworks/acctvn-220824145028554777/subnets/acctsub-220824145028554777 has reference to Basic SKU Public IP address or Load Balancer /subscriptions/*******/resourceGroups/acctestRG-OVMSS-220824145028554777/providers/Microsoft.Network/publicIPAddresses/TestPublicIPConfiguration-5c526597." Details=[]
with azurerm_subnet_nat_gateway_association.example,
on terraform_plugin_test.tf line 47, in resource "azurerm_subnet_nat_gateway_association" "example":
47: resource "azurerm_subnet_nat_gateway_association" "example" {
--- FAIL: TestAccOrchestratedVirtualMachineScaleSet_basicPublicIP (237.74s)
FAIL